.github/update-authors.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+// original comes from https://github.com/nodejs/node/blob/master/tools/update-authors.js
+
+// Usage: tools/update-author.js [--dry]
+// Passing --dry will redirect output to stdout rather than write to 'AUTHORS'.
+"use strict";
+const { spawn } = require("child_process");
+const fs = require("fs");
+const readline = require("readline");
+
+const log = spawn(
+  "git",
+  // Inspect author name/email and body.
+  ["log", "--reverse", "--format=Author: %aN <%aE>\n%b"],
+  {
+    stdio: ["inherit", "pipe", "inherit"],
+  }
+);
+const rl = readline.createInterface({ input: log.stdout });
+
+let output;
+if (process.argv.includes("--dry")) {
+  output = process.stdout;
+} else {
+  output = fs.createWriteStream("AUTHORS");
+}
+
+output.write("# Authors ordered by first contribution.\n\n");
+
+const seen = new Set();
+
+// exclude emails from <ROOT>/AUTHORS file
+const excludeEmails = ["bot@renovateapp.com"];
```

Comment:
This wasn't very successful it seems. The following are still in the list;
renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Renovate Bot <bot@renovateapp.com>

Reply:
that's the private email feature from github :)

Reply:
> that's the private email feature from github :)

Yeahh i know

Reply:
Okay i will using regex for `renovate[bot]@users.noreply.github.com` so we will make sure not to hardcoded every random numbers like `29139614+`